association/models.py

```diff
@@ -48,18 +48,18 @@ class AssociationStartDate(ComplexField):
     object_ref = models.ForeignKey('Association')
     value = ApproximateDateField()
     field_name = _("First Cited Date")
-    shortcode = 'u_aofcd'
-    spreadsheet_field_name = 'unit:area_ops_first_cited_date'
+    shortcode = 'u_lfcd'
```

Comment:
Spreadsheet field names and shortcodes should match this sheet: https://docs.google.com/spreadsheets/d/1-U_pVNDWFIlG8jLZLVujmV_FZ0H4IsbtfCeRiM8HeNQ/edit?usp=sharing